tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -230,6 +305,10 @@ def tap_options(self):
         Wait(self.marionette).until(expected.element_displayed(
             Wait(self.marionette).until(expected.element_present(*self._option_menu_locator))))
 
+    def tap_picture_preview(self):
+        self.marionette.find_element(*self._thumbnail_button_locator).tap()
+	Wait(self.marionette).until(lambda m: self.is_image_preview_visible is True)
```

Comment:
Nit: No need for `is True`, here. Methods like `self.is_something` imply that a boolean is returned